packages/db-ra-data-rest/src/index.ts

```diff
@@ -0,0 +1,153 @@
+import { stringify } from "query-string";
+import { fetchUtils, DataProvider } from "ra-core";
+
+/**
```

Comment:
Please include the original License text from Giacomo here.

Reply:
as above, the db-ra-data-rest folder will be removed, so ignore it please... for this reason I left my PR in draft.